package.json

```diff
@@ -17,6 +17,17 @@
             "url": "https://github.com/notanengineercom"
         }
     ],
+    "repository": {
+        "type" : "git",
+        "url" : "https://github.com/ffMathy/FluffySpoon.JavaScript.Testing.Faking.git"
+    },
+    "keywords": ["NSubstitute", "mocking", "test", "testing", "faking"],
+    "bugs": {
+        "url": "https://github.com/ffMathy/FluffySpoon.JavaScript.Testing.Faking/issues"
+    },
+    "engines": {
+        "node": ">=8"
```

Comment:
I noticed that you're running GitHub actions for Node 8, 10, and 12, so this seemed like a reasonable addition. Unless the user of the package has `engine-strict` mode turned on locally, this will only produce a warning.